notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2224 @@
+{
```

Comment:
<blockquote>To train a custom model, you perform two steps: 1) create a custom training job, and 2) run the job. </blockquote><p>To train a custom model, you perform two steps: </p><p>1) Create a custom training job</p><p>2) Run the job.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>done</p>